cmake/onnxruntime_providers.cmake

```diff
@@ -77,6 +77,17 @@ if(HAS_DEPRECATED_COPY)
   set_source_files_properties("${ONNXRUNTIME_ROOT}/core/providers/cpu/tensor/onehot.cc" PROPERTIES COMPILE_FLAGS -Wno-deprecated-copy)
   set_source_files_properties("${ONNXRUNTIME_ROOT}/core/providers/cpu/tensor/where_op.cc" PROPERTIES COMPILE_FLAGS -Wno-deprecated-copy)
 endif()
+
```

Comment:
LGTM but it's not very useful (to me).
We have another flag, onnxruntime_BUILD_FOR_NATIVE_MACHINE. 
If perf is a concern, build from source and enable is flag. It will enable msse4.1, if your processor supports it. (I guess)

Reply:
Yes I am aware of this flag... Infact when we added gemmlowp we did not add sse4.1 here with the intention to use onnxruntime_BUILD_FOR_NATIVE_MACHINE when necessary. 

For some partners who take dependency on pre built packages this is better option than building from source so adding this now.